### PR TITLE
docs: add README and CONTRIBUTE

### PR DESCRIPTION
**Pull request by redeboer**
Requires  https://github.com/ComPWA/tensorwaves/pull/4 and https://github.com/ComPWA/tensorwaves/pull/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# TensorWaves
+
+TensorWaves is a Python package for doing Partial Wave Analysis with [TensorFlow](https://www.tensorflow.org/) as computational backend. The package originates from [pycompwa](https://compwa.github.io/) (see in particular the [expert system](./tensorwaves/expertsystem)), which did its computations through ([ComPWA](https://github.com/ComPWA/ComPWA)) ― ComPWA's function tree resembles the structure of a neural network and can therefore be replaced by the tools that TensorFlow already offers. The decision to completely migrate ComPWA to TensorFlow has been further inspired by [TensorFlowAnalysis](https://gitlab.cern.ch/poluekt/TensorFlowAnalysis) and [zfit](https://github.com/zfit/zfit/).
```
Comment (spflueger) [inline README.md]: >resembles structure of a neural network
I would change that to `is similar to tensorflow graphs`

Comment (spflueger) [inline README.md]: >[pycompwa](https://compwa.github.io/) 

That links seems wrong as well

File: README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# TensorWaves
+
```
Comment (spflueger) [inline README.md]: Mention this is work in progress and is currently instable

File: README.md (hunk 1 above)
Comment (redeboer) [reply README.md]: What should the link be?

Comment (spflueger) [reply README.md]: nvm its fine, just looks strange

**Pull request merged**